src/table/components/FooterWrapper.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import Paper from '@mui/material/Paper';
+
+const Portal = ({ children, target }) => ReactDOM.createPortal(children, target);
+
+export default function FooterWrapper({ children, theme, footerContainer }) {
+  const paperTablePaginationStyle = {
+    height: 48,
```

Comment:
Did some css changes. All of this is still arbitrary, but this is a qualified guess to what UX wants. The input and buttons are 32px high, then adding 8px padding and we get 48.